discopop_explorer/pattern_detectors/task_parallelism/tp_utils.py

```diff
@@ -431,9 +431,7 @@ def __get_word_prior_to_bracket(string):
         string = string[-1]
         return string
 
-    called_function_name_contained = False
-    if called_function_name is None:
-        called_function_name_contained = True
+    called_function_name_contained = called_function_name is None
```

Comment:
i think we can simplify further and get rid of the variable entirely, but it is late and my brain is not 100% sure